tests/contrib/pylons/test_pylons.py

```diff
@@ -752,20 +754,21 @@ def test_request_method_post_200(self):
         assert spans[0].get_tag("http.method") == "POST"
 
     def test_pylon_path_params(self):
-        self.tracer._appsec_enabled = False
-        self.tracer.configure(api_version="v0.4")
-        self.app.get("/path-params/2022/july/")
+        with override_global_config(dict(_appsec_enabled=True)):
```

Comment:
Not sure this makes sense, shouldn't it be `_appsec_enabled=False`?

Reply:
No It shouldn't, because if appsec_enabled is False, we don't set the context and this assert will fail:

```
            assert root_span.get_tag("_dd.appsec.json") is None
            path_params = _context.get_item("http.request.path_params", span=root_span)

            assert path_params["month"] == "july"
            assert path_params["year"] == "2022"
```